import/marc.properties

```diff
@@ -19,6 +19,10 @@ language = 008[35-37]:041a:041d:041h:041j, language_map.properties
 # the record by examining the leader, 007, 008, 086, 111/711, 245h and 502.
 format = custom, getFormats, format_map.properties, unique
 
+alt_title = LNK245ab, first
+alt_title_sub = LNK245b, first
+alt_title_short = LNK245a, first
```

Comment:
When working on this, I felt like it was really nice to have all these fields named with a _prefix_ rather than a _suffix_. This allows them to all be grouped and sorted together in one place. I felt like that was nice and made for an easy way to keep track of them. That being said, I don't know if other's would rather use a _suffix_ and have them interfile with their corresponding fields. I don't feel strongly about this, so I'm happy to change it. Since, I'm assuming that `multivalued` fields will just get the fields added to their configuration, this "separation" of linked fields with prefixes, will not be pure anyhow. I will note, however, that if we use a suffix it cannot be `_alt` because `title_alt` already exists.

Reply:
I don't have strong opinions about this, and the best option may naturally present itself as we explore further... but I do feel strongly that we should avoid having fields names both `alt_title` and `title_alt` -- nobody is going to understand the difference between those, and it will cause confusion. Looks like the current `title_alt` field is used for alternate title forms and secondary titles, while of course 880 is for alternate renderings of the same title. A perfect prefix or suffix for capturing this distinction is not leaping to my mind, but I'll keep pondering it...

Reply:
...this is perhaps too verbose, but you're using AlternateDisplay as the prefix for method names in the record driver, so would `alt_display_` be a sensible prefix for these fields?

Reply:
I agree that the best option might present itself as we explore further but `alt_display_` is reasonable...